great_expectations/expectations/expectation.py

```diff
@@ -3337,31 +3235,14 @@ class MulticolumnMapExpectation(BatchExpectation, ABC):
         "condition_parser": None,  # we expect this to be explicitly set whenever a row_condition is passed
         "mostly": 1,
         "ignore_row_if": "all_values_are_missing",
-        "result_format": "BASIC",
-        "include_config": True,
-        "catch_exceptions": True,
```

Comment:
Both these fields are irrelevant moving forward - I'm going to delete them as I go